go/vt/mysqlctl/mysqld.go

```diff
@@ -460,6 +460,10 @@ func cleanupLockfile(socket string, ts string) error {
 		log.Errorf("%v: error parsing pid from lock file: %v", ts, err)
 		return err
 	}
+	if os.Getpid() == p {
+		log.Infof("%v: lock file at %s is ours, removing it", ts, lockPath)
```

Comment:
I'd take the opportunity to rename `ts` as `ts == *topo.Server` in the code base. :-) 

No idea why that was chosen for:
```
ts := fmt.Sprintf("Mysqld.Start(%v)", time.Now().Unix())
```

time start, I guess? Up to you if you want to change it.